File: flight-analysis-server/src/main/java/czu/bigdata/flightAnalysis/controller/TestController.java
package czu.bigdata.flightAnalysis.controller;

import com.alibaba.fastjson.JSONArray;
import czu.bigdata.flightAnalysis.service.WordCountService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import javax.annotation.Resource;

@RestController
public class TestController {
    @Resource
    private final WordCountService wordCountService;

    public TestController( WordCountService wordCountService) {
        this.wordCountService = wordCountService;
    }

    @GetMapping("/")
    public JSONArray wordCount() {
        return wordCountService.wordCount();
    }
}

File: flight-analysis-server/src/main/scala/czu/bigdata/flightAnalysis/controller/TestController.scala
//package czu.bigdata.flightAnalysis.controller
//
//import com.alibaba.fastjson.JSONArray
//import czu.bigdata.flightAnalysis.service.WordCountService
//import org.springframework.web.bind.annotation.GetMapping
//import org.springframework.web.bind.annotation.RequestParam
//import org.springframework.web.bind.annotation.RestController
//import javax.annotation.Resource
//
//
//@RestController class TestController(@Resource private val wordCountService: WordCountService) {
//  @GetMapping(Array("/wordCount")) def wordCount(@RequestParam filePath: String): JSONArray = wordCountService.wordCount(filePath)
//}
